src/wgquickprocess.cpp

```diff
@@ -20,10 +20,13 @@ Logger logger(
     "WgQuickProcess");
 
 QString scriptPath() {
+  QDir appPath(QCoreApplication::applicationDirPath());
```

Comment:
What about if in `src.pro` we add:
```
DEFINES += MVPN_DATA_PATH=\\\"$${USRPATH}/share/mozillavpn\\\"
```

And here we do:
```
QDir appPath(MVPN_DATA_PATH);
return appPath.filePath("helper.sh");
```

Reply:
Given that this helper file will be going away shortly I don't think we need to.

Reply:
Fixed in 1acbb8f